Add RediSearch Jedis quick start (#2996)
redis/jedis: README.md

JedisPool pool = new JedisPool("localhost", 6379);
```

> Once you have a `JedisPool` instance, you can use a

Reviewer: ```
With a `JedisPool` instance, you can use a
```

> Once you have a `JedisPool` instance, you can use a

Author: ```suggestion
With a `JedisPool` instance, you can use a
```

}
```

`Jedis` instances implement most Redis commands. See the
[Jedis Javadocs](https://www.javadoc.io/doc/redis.clients/jedis/latest/redis/clients/jedis/Jedis.html)

> for a complete list of supported commands.

Reviewer: ```
for the complete list of supported commands.
```

> for a complete list of supported commands.

Author: ```suggestion
for the complete list of supported commands.
```
